pkg/operation/seed/components.go

```diff
@@ -225,10 +230,27 @@ func defaultExternalAuthzServer(
 		return nil, err
 	}
 
-	return extauthzserver.NewExtAuthServer(
+	extAuthServer := extauthzserver.NewExtAuthServer(
 		c,
 		v1beta1constants.GardenNamespace,
 		image.String(),
 		3,
-	), nil
+	)
+
+	if gardenletfeatures.FeatureGate.Enabled(features.ManagedIstio) {
+		return extAuthServer, nil
+	}
+
+	vpnSeedDeployments := &appsv1.DeploymentList{}
+	if err := c.List(ctx, vpnSeedDeployments, client.MatchingLabels(map[string]string{v1beta1constants.LabelApp: v1beta1constants.DeploymentNameVPNSeedServer}), client.Limit(1)); err != nil {
```

Comment:
Nit: What about doing a metadata-only list here?

Reply:
Good idea 👍 Done in 2a664004c66669f0b403013646f8f8533d358dec.